WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -9186,6 +9191,7 @@
 		E1B34C091CCDFFCE00889709 /* Credentials */ = {
 			isa = PBXGroup;
 			children = (
+				29B97317FDCFA39411CA2CEA /* Resources */,
```

Comment:
@ScoutHarris was this move intentional?  Seems like Resources shouldn't be a child of Credentials.

Reply:
Gah! Nice catch! It's been fixed with https://github.com/wordpress-mobile/WordPress-iOS/pull/13177.